### PR TITLE
TCP test setup to vary based on build tag

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

**Comment by fgiorgetti** [comment]
This PR allows the TCP performance test to run differently based on provided build tags.
When using `-tags=integration` (as done by CI), it will set debug options to the router (gdb, trace+).
When `-tags=performance` is provided, then router debug options won't be set.

File: test/integration/performance/tcp/tcp.go [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+	ENV_SKUPPER_SITES              = "SKUPPER_SITES"
+	ENV_SKUPPER_MAX_FRAME_SIZE     = "SKUPPER_MAX_FRAME_SIZE"
+	ENV_SKUPPER_MAX_SESSION_FRAMES = "SKUPPER_MAX_SESSION_FRAMES"
+	ENV_SKUPPER_MEMORY             = "SKUPPER_MEMORY"
+	ENV_SKUPPER_CPU                = "SKUPPER_CPU"
+)
+
+func TestMain(m *testing.M) {
+	log.Println("BATATA")
```
Comment (granzoto) [inline test/integration/performance/tcp/tcp.go:35]: Banana works better here !!

Comment (fgiorgetti) [reply test/integration/performance/tcp/tcp.go]: I knew I should have used a fruit instead. :)

**Review by granzoto — APPROVED** [review]
LGTM, Nice change

**Review by grs — APPROVED** [review]

**Review by nluaces — APPROVED** [review]

**Pull request merged**